src/index.js

```diff
@@ -197,20 +195,24 @@ class Radar {
 
     Trips.updateTrip(tripOptions, TRIP_STATUS.CANCELED)
       .then((response) => {
-        // clear tripOptions
-        Storage.removeItem(Storage.TRIP_OPTIONS);
+        Radar.clearTripOptions();
         callback(null, { trip: response.trip, events: response.events, status: STATUS.SUCCESS }, response);
       })
       .catch(handleError(callback));
   }
 
   static setTripOptions(tripOptions) {
     if (!tripOptions) {
+      Radar.clearTripOptions();
       return;
     }
     Storage.setItem(Storage.TRIP_OPTIONS, JSON.stringify(tripOptions));
```

Comment:
Would 

```
if (!tripOptions) {
    Storage.setItem(Storage.TRIP_OPTIONS, JSON.stringify(tripOptions));
}
```

have the same effect, or does explicitly calling `Storage.removeItem()` have some side effects that need to happen?